assets/css/main.css

```diff
@@ -23,3 +25,8 @@ html {
   height: 100vh;
   overflow: hidden;
 }
+
```

Comment:
Why is the style for `.FlexContainer` in `main.css` and not in `eventform.css`?

Reply:
I though it may be generalised since we se it often on frontend.